src/connector/src/source/kinesis/source/message.rs

```diff
@@ -25,6 +26,7 @@ pub struct KinesisMessage {
 }
 
 impl From<KinesisMessage> for SourceMessage {
+    // not in use
```

Comment:
I guess we can remove `KinesisMessage` at all?

Reply:
yes, ,let me remove it

Reply:
Oh, source trait requires one connector-spec message struct. Let me remove the components and keep the struct itself